src/Configuration/ConfigDefinition.php

```diff
@@ -59,8 +59,8 @@ class ConfigDefinition implements ConfigurationInterface
     const PHP_WORKER_NAME = 'name';
 
     /** Default values */
-    const LOG_DIR_DEFAULT = 'var/log';
-    const CACHE_DIR_DEFAULT = 'var/cache/disqontrol';
+    const LOG_DIR_DEFAULT = '/var/log';
+    const CACHE_DIR_DEFAULT = '../cache/disqontrol';
```

Comment:
I think that the system path `/tmp/disqontrol` will be a better default path. We don't know structure of the "parent" application